src/js/tooltip.js

```diff
@@ -458,23 +481,23 @@ class Tooltip {
 		if (point < 0) {
 			return true;
 		}
-		if (axis === 'y' && point > document.body.offsetHeight) {
+		if (axis === 'y' && point > document.documentElement.clientHeight) {
```

Comment:
Can we have a comment explaining why we are using this instead of `document.body.offsetHeight`